modal/object.py

```diff
@@ -199,7 +199,7 @@ def deps(self) -> Callable[..., List["_Object"]]:
         """mdmd:hidden"""
         return self._deps if self._deps is not None else lambda: []
 
-    async def resolve(self):
+    async def resolve(self, client: Optional[_Client] = None):
```

Comment:
this is a minor code change slightly outside the scope of the PR – I needed it in a test